Parser.py

```diff
@@ -0,0 +1,98 @@
+from Mailer import Mailer
+import urllib.request
+
+mailer = Mailer()
+
+class Parser:
+    # Load data from enviromental variables, set the parse URL and load all the parsed URLs
+    def __init__(self, parseURL):
+        print('Initializing parser')
+        self.ads = []
+        file = open('url_list.txt', 'r')
+        lines = file.readlines()
+        file.close()
+        for line in lines:
+            url = line.replace('\n', '')
```

Comment:
.replace() method is used for replacing a substring, you want to remove it.
```suggestion
            url = line.strip('\n')
```